src/dropWhile.js

```diff
@@ -1,6 +1,7 @@
 var _curry2 = require('./internal/_curry2');
 var _dispatchable = require('./internal/_dispatchable');
 var _xdropWhile = require('./internal/_xdropWhile');
+var slice = require('./slice');
```

Comment:
But if using Array.prototype.slice, the returned value will always be an array. Then how to let dropWhile compatible with String?